test/test_tidsserie.py

```diff
@@ -0,0 +1,61 @@
+import fire
+from fire.api import FireDb
+from fire.api.model import (
+    Punkt,
+    PunktGruppe,
+    Tidsserie,
+    EventType,
+)
+
+
+def test_tidsserie(firedb, punkt, punktgruppe, srid, koordinatfabrik):
+    """Test oprettelse af en tidsserie."""
+    # Tidsserie med punktgruppe og jessenkoordinat
+    ts1 = Tidsserie(
+        punkt=punkt,
+        punktgruppe=punktgruppe,
+        jessenkoordinat=koordinatfabrik(),
+        referenceramme="FIRE",
+        srid=srid,
+        koordinater=[koordinatfabrik() for _ in range(5)],
+    )
+
+    firedb.session.flush()
+    assert isinstance(ts1, Tidsserie)
+
+    # Tidsserie uden punktgruppe og jessenkoordinat
+    ts2 = Tidsserie(
+        punkt=punkt,
+        referenceramme="FIRE",
+        srid=srid,
+        koordinater=[koordinatfabrik() for _ in range(5)],
+    )
+
+    firedb.session.flush()
+    assert isinstance(ts2, Tidsserie)
```

Comment:
Også her.

Min tilgang ville være at læse oprettede data ud fra databasen igen for at kontrollere, at de er oprettet korrekt nede i databasen.

Men jeg kan måske odt overbevises, hvis der er sådan, at SQLAlchemy har nogle tricks i ærmet, jeg ikke kender. Det lader ihvertfald til, at koden kun undersøger, om noget er dét, det lige blev oprettet som, altså nærmest en tautologisk assert.

Om det er hentet ud fra databasen eller ej, skal det vel som minimum testes, at indholdet af instansens medlemmer er, som de blev sat ved instantieringen.